### PR TITLE
Append AUX mixer for HITL simulation

### PR DESCRIPTION
**Pull request by bartslinger**
For HITL simulation, I need to capture all outputs of the pixhawk, including the AUX ones. With this pull-request, it basically works like this:

1. Main mixer is loaded to pwm_out_sim
2. Aux mixer is *appended* to pwm_out_sim

The append function is basically the same as load, but it does not reset the mixer first.

### CODE REVIEW

File: ROMFS/px4fmu_common/init.d/rc.interface [hunk 1]
```diff
@@ -138,7 +138,18 @@ then
...
+			then
+				if mixer append $OUTPUT_DEV $MIXER_AUX_FILE
+				then
+					echo "INFO  [init] Mixer: $MIXER_AUX_FILE appended to $OUTPUT_DEV"
+				else
+					echo "ERROR [init] Error appending mixer: $MIXER_AUX_FILE"
+					echo "ERROR [init] Could not append mixer: $MIXER_AUX_FILE" >> $LOG_FILE
+				fi
+			fi
+			if [ -e $OUTPUT_AUX_DEV -a $OUTPUT_MODE != hil ]
```
Comment (LorenzMeier) [inline ROMFS/px4fmu_common/init.d/rc.interface]: Have you tested this line on real hardware?

Comment (bartslinger) [inline ROMFS/px4fmu_common/init.d/rc.interface]: Yes I'm running this on a Pixhawk, so when I use the airframe without HIL, the console shows:
```
INFO  [init] Mixer: /etc/mixers/octo_vtol.main.mix on /dev/pwm_output0 
INFO  [init] Mixer: /etc/mixers/vtol_tailsitter_AAEE.aux.mix on /dev/pwm_output1 
```

Then I added this at the bottom of the airframe:
`set HIL yes`

Which gives:
```
INFO  [init] Mixer: /etc/mixers/octo_vtol.main.mix on /dev/pwm_output0 
INFO  [init] Mixer: /etc/mixers/vtol_tailsitter_AAEE.aux.mix appended to /dev/pwm_output0 
```

Comment (LorenzMeier) [inline ROMFS/px4fmu_common/init.d/rc.interface]: Nice. Will merge then now.

Comment (bartslinger) [inline ROMFS/px4fmu_common/init.d/rc.interface]: Thanks!

**Pull request merged**